tstest/integration/vms/vms_test.go

```diff
@@ -80,6 +80,55 @@ func (d *Distro) InstallPre() string {
 	return ""
 }
 
+func TestDownloadImages(t *testing.T) {
```

Comment:
Sorry, this should still be guarded by the runVMTests flag, otherwise everybody (and GitHub actions) will try to download a bazillion GB.

If you want to add a new flag `--allow-external-network` that's fine too, but make sure that `TestVMIntegrationEndToEnd` also respects it if so. I'd make `fetchDistro` check it and t.Skip there.